hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/test/java/org/apache/hadoop/yarn/server/router/webapp/TestFederationInterceptorREST.java

```diff
@@ -142,6 +154,23 @@ public void setUp() {
       Assert.fail();
     }
 
+    try {
+      for (SubClusterId subCluster : subClusters) {
+        SubClusterInfo subClusterInfo = stateStoreUtil.querySubClusterInfo(subCluster);
+        interceptor.getOrCreateInterceptorForSubCluster(
+            subCluster, subClusterInfo.getRMWebServiceAddress());
+      }
+    } catch (YarnException e) {
```

Comment:
Why isn't enough to surface the exception?

Reply:
Thanks for your help reviewing the code, I will fix it.